app/jobs/export_job.rb

```diff
@@ -5,34 +5,56 @@ class ExportJob < ApplicationJob
   queue_as :exports
   include ImportExport
 
-  # Limits number of Patient records to be considered for a single exported file to handle maximum file size limit.
-  # Adds additional files as needed if exceeds batch size.
-  OUTER_BATCH_SIZE = ENV['EXPORT_OUTER_BATCH_SIZE']&.to_i || 10_000
-
   # Inner batch size limits number of Patient records details help in memory at once before writing to file.
-  INNER_BATCH_SIZE = ENV['EXPORT_INNER_BATCH_SIZE']&.to_i || 500
+  INNER_BATCH_SIZE = ENV.fetch('EXPORT_INNER_BATCH_SIZE', 500).to_i unless const_defined?(:INNER_BATCH_SIZE)
 
   def perform(config)
     # Get user in order to query viewable patients
     user = User.find_by(id: config[:user_id])
     return if user.nil?
 
     # Delete any existing downloads of this type
-    user.downloads.where(export_type: config[:export_type]).delete_all
+    user.downloads.where(export_type: config[:export_type]).destroy_all
 
     # Extract data
     data = config[:data]
     return if data.nil?
 
     # Construct export
+    downloads = []
+
     patients = patients_by_query(user, data.dig(:patients, :query) || {})
-    files = write_export_data_to_files(config, patients, OUTER_BATCH_SIZE, INNER_BATCH_SIZE)
-    return unless files.present?
+    files = write_export_data_to_files(config, patients.reorder(''), INNER_BATCH_SIZE)
```

Comment:
The original reorder came from https://github.com/SaraAlert/SaraAlert/commit/e2f933f66230dd6b237acd0901263c3b54301f8d#diff-36aa606b2baf34b6cc66a195a7b4e92ebe659b6858b2863523c608d08d2d86abR33 but was removed https://github.com/SaraAlert/SaraAlert/commit/9e14c3c5763477000d88c3741989207d368a629c#diff-36aa606b2baf34b6cc66a195a7b4e92ebe659b6858b2863523c608d08d2d86abL33

@timwongj can it be removed?

Reply:
I think @kylietmo originally put this in here because the ordering would get messed up during batching, but that was when we had inner and outer batches, so I'm not sure if that's necessary anymore

Reply:
Hey!  Yeah it's there because when I first added the batching I found it wouldn't order by ID anymore (as is expected) without it, as in tests would actually fail. If there is any batching occuring still (even just one layer of it), it may still be necessary as it seemed to be a result of the find_in_batches (or whichever one we used) method. Should be able to test by removing and watching the system tests.

Reply:
thanks @kylietmo !

Reply:
Thanks for the update. We've been running without batching in the updated exports for two weeks at this point and I don't think anyone has raised an issue.